editor/src/clj/editor/cljfx_form_view.clj

```diff
@@ -913,6 +913,26 @@
      :cell-value-factory (fn/partial table-cell-value-factory path)
      :cell-factory (fn/partial table-cell-factory column (dissoc edit :value))}))
 
+(defn custom-table-resize-policy []
```

Comment:
It seems we can use `def` instead of `defn` here, since we only really need one constant instance.

Reply:
It makes sense, yes. thx